test/dataExtension.test.js

```diff
@@ -17,10 +17,11 @@ describe('dataExtension', () => {
             // THEN
             // get results from cache
             const result = cache.getCache();
+            console.log('result:::::', result);
```

Comment:
should we not at least be adding one or two tests for these key changes?

Reply:
created #342 for that purpose. would be great if you could tackle that because I'm having a really hard time a) understanding mocha still and b) mocking your SDK